parent/pom.xml

```diff
@@ -260,7 +260,7 @@
                 <pluginRepository>
                     <id>sonatype-nexus-snapshots</id>
                     <name>Sonatype Nexus Snapshots</name>
-                    <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+                    <url>https://jakarta.oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Same comment as above...  Any reason why we shouldn't use the existing sonatypeOssDistMgmtSnapshotsUrl property?